spec/spec_helper.rb

```diff
@@ -19,6 +19,8 @@
 # in ./support/ and its subdirectories.
 Dir["#{File.dirname(__FILE__)}/support/**/*.rb"].sort.each { |f| require f }
 
+RuboCop::Cop::Registry.global.freeze
```

Comment:
Might be a good idea to add a comment describing why we freeze it in the specs.

Reply:
Agreed, I moved the commit message just after the freeze 👍